src/sagemaker/remote_function/invoke_function.py

```diff
@@ -89,12 +95,26 @@ def main():
         s3_kms_key = args.s3_kms_key
         run_in_context = args.run_in_context
 
+        hmac_key = os.getenv("SECRET_KEY")
```

Comment:
nit: we should give this env variable a unique name - something the user will not overwrite